code/modules/surgery/organs/tongue.dm

```diff
@@ -239,6 +239,7 @@
 /obj/item/organ/tongue/abductor/examine(mob/M)
 	. = ..()
 	if(HAS_TRAIT(M, TRAIT_ABDUCTOR_TRAINING) || HAS_TRAIT(M.mind, TRAIT_ABDUCTOR_TRAINING) || isobserver(M))
```

Comment:
```suggestion
	if(HAS_TRAIT(M, TRAIT_ABDUCTOR_TRAINING) || (M.mind && HAS_TRAIT(M.mind, TRAIT_ABDUCTOR_TRAINING)) || isobserver(M))
```

Apologies for this change request, but while you're in the area could you make this change. This'll cause a runtime
if a roundstart observer examines an abductor tongue, since they don't have minds.